docs/layers/arc-layer.md

```diff
@@ -8,14 +8,16 @@
 
 The Arc Layer renders raised arcs joining pairs of source and target points,
 specified as latitude/longitude coordinates.
+[Source](https://github.com/uber/deck.gl/tree/master/src/layers/core/arc-layer)
```

Comment:
We previously decided to put source links at the bottom, partially to be consistent with THREE.js, gl-matrix etc, and because it is an additional level of detail, together with remarks etc.

Is this important? Do we need to discuss?